docs/user-guide/aes-operator.md

```diff
@@ -0,0 +1,127 @@
+
+1. To create the `AmbassadorInstallation` Custom Resource schema, use the following YAML as your guideline: [https://github.com/datawire/ambassador-operator#the-operator-custom-resource-cr](https://github.com/datawire/ambassador-operator#the-operator-custom-resource-cr).
+2. Save that file as `amb-install.yaml`
+3. Edit the `amb-install.yaml` and optionally complete configurations such as Version constraint or UpdateWindow:
+4. Finally, apply your `AmbassadorInstallation` CRD to the AES Operator schema
+   with the following command: `kubectl apply -n ambassador -f amb-install.yaml`
+
+### Configuration for the Ambassador Edge Stack
+
+After the initial installation of Ambassador, the Operator will check for updates every 24 hours and delay the update until the Update Window allow the update to proceed. It will use the Version Syntax for determining if any new release is acceptable. When a new release is available and acceptable, the Operator will upgrade the Ambassador installation.
+
+### Version Syntax and Update Window
+
+To specify version numbers, use SemVer for the version number for any level of
+precision. This can optionally end in `*`.  For example:
+
+  * `1.0` = exactly version 1.0
+  * `1.1` = exactly version 1.1
+  * `1.1.*` = version 1.1 and any bug fix versions 1.1.1, 1.1.2, 1.1.3, etc.
+  * `2.*` = version 2.0 and any incremental and bug fix versions 2.0, 2.0.1, 2.0.2, 2.1, 2.2, 2.2.1, etc.
+  * `*` = all versions.
+  * `3.0-ea` = version 3.0-ea1 and any subsequent EA releases on 3.0. Also selects the final 3.0 once the final GA version is released.
+  * `4.*-ea` = version 4.0-ea1 and any subsequent EA release on 4.0. This also selects:
+      * the final GA 4.0.
+      * any incremental and bug fix versions 4.* and 4
+      * the most recent 4.* EA release (i.e., if 4.0.5 is the last GA version and
+        there is a 4.1-EA3, then this selects 4.1-EA3 over the 4.0.5 GA).
+
+Read more about SemVer [here](https://github.com/Masterminds/semver#basic-comparisons).
+
+`updateWindow` is an optional item that will control when the updates can take
+place. This is used to force system updates to happen late at night if that’s
+what the sysadmins want.
+
+There can be any number of `updateWindow` entries (separated by commas).
+Never turns off automatic updates even if there are other entries in the
+comma-separated list. Never is used by sysadmins to disable all updates during
+blackout periods by doing a `kubectl` apply or using our Edge Policy Console to
+set this.
+
+Each `updateWindow` is in crontab format (see https://crontab.guru/) Some
+examples of `updateWindow` are:
+
+* `0-6 * * * SUN`: every Sunday, from 0am to 6am
+* `5 1 * * *`: every first day of the month, at 5am
+
+The Operator cannot guarantee minute time granularity, so specifying a minute in the crontab expression can lead to some updates happening sooner/later than expected.
+
+### Install via Helm Chart
```

Comment:
I would move this section to the "install the operator" section, because it is just another way of installing it